vyxal/elements.py

```diff
@@ -2390,7 +2390,9 @@ def is_divisible(lhs, rhs, ctx):
     def helper(lhs, rhs):
         ts = vy_type(lhs, rhs)
         return {
-            (NUMBER_TYPE, NUMBER_TYPE): lambda: int(lhs % rhs == 0),
+            (NUMBER_TYPE, NUMBER_TYPE): lambda: int(lhs % rhs == 0)
```

Comment:
If we were golfing I'd suggest `rhs and int(lhs ...)` but that'd make this unclear :(